src/components/account/CreateSetButton.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <b-card
+    class="text-center default-card content-card create-card"
+    no-body
+    data-qa="create new gallery card"
+    @click="clickCreateSet"
+  >
+    <div class="card-inner d-flex justify-content-center align-items-center">
+      <span class="icon-ic-add" />
+      <b-card-body data-qa="card body">
+        <b-card-title title-tag="div">
+          <span>
```

Comment:
What is the purpose of the span here? 
Could the span be removed or could the title-tag be set to a span instead of a div directly?

Reply:
The span is also (unnecessarily?) there on the ContenCard titles, so we just copied it from there. I removed the span for the CreateSet and that seems fine indeed.